compiler/rustc_hir/src/intravisit.rs

```diff
@@ -961,8 +961,9 @@ pub fn walk_where_predicate<'v, V: Visitor<'v>>(
     visitor: &mut V,
     predicate: &'v WherePredicate<'v>,
```

Comment:
Do you mind destructuring `predicate`, so this file stops compiling when we modify HIR.